Dockerfile

```diff
@@ -84,6 +84,7 @@ COPY . .
 # And compile the project
```

Comment:
btw, docker build fails on the `curl` line 57 unless I add the -k flag (ignore certificates), but also I've been working from a WSL on a Windows machine so it could be a local issue.